shared/actions/chat.js

```diff
@@ -298,8 +299,12 @@ function * _setupNewChatHandler (): SagaGenerator<any, any> {
 const followingSelector = (state: TypedState) => state.config.following
 
 function * _loadInbox (): SagaGenerator<any, any> {
-  // $ForceType
-  const inbox: GetInboxAndUnboxLocalRes = yield call(localGetInboxAndUnboxLocalRpcPromise, {params: {}})
+  const inbox: GetInboxAndUnboxLocalRes = ((yield call(localGetInboxAndUnboxLocalRpcPromise, {param: {
+    query: {
+      status: Object.keys(CommonConversationStatus).filter(k => !['ignored', 'blocked'].includes(k)).map(k => CommonConversationStatus[k]),
```

Comment:
This is weird... We're passing in all statuses here that aren't ignored or blocked?

Can we add a `exceptStatus: [CommonConversationStatus.IGNORED_2, CommonConversationStatus.BLOCKED_3]` to the query?